docker-compose/cassandra-3.11/docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
     networks:
       - stargate
     ports:
-      - 9044:9042
+      - 9042:9042
```

Comment:
there was a discussion around this in [[stargate/docker]()](https://github.com/stargate/docker-images/pull/95#discussion_r795954868).. now when I think about this: 
* `9042` (default) should be mapped to Stargate.. 
* and I am not sure if we need to expose the CQL port for C, if yes, then let's keep it `9044`? I mean we should want all CQL traffic to go through Stargate or?

Reply:
yeah, I kind of kept this convention from the original examples in `docker-images`. I assumed that exposing CQL port on one of the Cassandra nodes was for debugging purposes, but I agree with your comment. I'm not sure that this is a practice that we want to recommend.  Will remove this.